app/client/components/analytics.tsx

```diff
@@ -80,64 +81,77 @@ export const applyAnyOptimiseExperiments = () => {
 
 export const AnalyticsTracker = () => {
   const [cmpIsInitialised, setCmpIsInitialised] = useState<boolean>(false);
+  const [gaIsInitialised, setGaIsInitialised] = useState<boolean>(false);
 
-  if (typeof window !== "undefined" && window.ga) {
-    const queryParams = parse(window.location.href, true).query;
+  const initialiseGa = () => {
+    // Run self evoking GA script
+    runGaStub();
 
-    const INTCMP = queryParams.INTCMP;
+    if (typeof window !== "undefined" && window.ga) {
+      const queryParams = parse(window.location.href, true).query;
 
-    if (window.guardian) {
-      // tslint:disable-next-line:no-object-mutation
-      window.guardian.INTCMP = INTCMP;
+      const INTCMP = queryParams.INTCMP;
 
-      const abName = queryParams.abName;
-      const abVariant = queryParams.abVariant;
+      if (window.guardian) {
+        // tslint:disable-next-line:no-object-mutation
+        window.guardian.INTCMP = INTCMP;
+
+        const abName = queryParams.abName;
+        const abVariant = queryParams.abVariant;
+
+        if (abName && abVariant) {
+          // tslint:disable-next-line:no-object-mutation
+          window.guardian.abTest = {
+            name: abName,
+            variant: abVariant
+          };
+        }
+      }
 
-      if (abName && abVariant) {
+      if (window.dataLayer === undefined) {
         // tslint:disable-next-line:no-object-mutation
-        window.guardian.abTest = {
-          name: abName,
-          variant: abVariant
-        };
+        window.dataLayer = [];
       }
-    }
 
-    if (window.dataLayer === undefined) {
-      // tslint:disable-next-line:no-object-mutation
-      window.dataLayer = [];
+      window.ga("create", GA_UA, "auto");
+      window.ga("require", "GTM-M985W29");
```

Comment:
This doesn't appear to be one of our Google Tag Manager properties. Does anybody know why we require this?

Reply:
cc @twrichards ?